bio_market_store/store/views.py

```diff
@@ -165,16 +167,17 @@ def logout_view(request):
 @login_required
 def user_profile(request):
     user_profile_form = UserProfileForm(instance=request.user)
-    address, created = Address.objects.get_or_create(user=request.user)
-    address_form = AddressForm(instance=address)
     password_form = PasswordChangeForm(request.user)
 
+    # Initialize address formset
+    address_formset = AddressFormSet(instance=request.user)
+
     return render(
         request,
         "user_profile.html",
         {
             "user_profile_form": user_profile_form,
-            "address_form": address_form,
+            "address_formset": address_formset,

```

Comment:
If you change to `address_formset` why in .htmt you used ` {{ form.DELETE }} ....`?